app/Models/Bible/BibleFileset.php

```diff
@@ -195,15 +201,17 @@ public static function filesetTypeTextPlainAssociated(?string $fileset_id) : Bib
         }
 
         // E.g fileset_associated = NT and text plain = NTP
+        $query = self::subqueryFilesetTypeTextPlainAssociated($bible_id);
         $fileset = $query
-            ->whereLike('set_size_code', 'LIKE', '%'.$fileset_associated["set_size_code"].'%')
+            ->where('set_size_code', 'LIKE', '%'.$fileset_associated["set_size_code"].'%')
             ->first();
 
         if ($fileset && $fileset["id"]) {
             return $fileset;
         }
 
         // If no specific fileset is found, default to SIZE_COMPLETE (six character fileset)
+        $query = self::subqueryFilesetTypeTextPlainAssociated($bible_id);
```

Comment:
@bradflood , it has been fixed because in the old code the instance of variable query was including all constraints and when it had to execute the last query it had been build incorrectly. Then, I have created the method subqueryFilesetTypeTextPlainAssociated to get a fresh query instance for each constraint.